src/kernels/Conv_Winograd_v30_3_1_gfx9_fp16_dot2_edc_f3x2_stride2.inc

```diff
@@ -24,66 +24,109 @@
  *
  *******************************************************************************/
 
+.macro _v_add_f16_e32_1_gfx9x vdst:req, vsrc:req
+    .long  0x3E0000FF + (\vdst << 17) + (\vsrc << 9)
+    .long  0x00003C00
+.endm
+
+.macro _buffer_store_short vdata, vidx, srd, idx, mode
+#.if FORCE_CACHE_BYPASS_ON_STORE
+#    buffer_store_short \vdata, \vidx, \srd, \idx \mode sc0 sc1
+#.else
+    buffer_store_short \vdata, \vidx, \srd, \idx \mode
+#.endif
+.endm
```

Comment:
`FORCE_CACHE_BYPASS_ON_STORE` is needed for gfx940/gfx941 ASICs, otherwise there would be functional correctness issues.